officemanager/department/views/views.py

```diff
@@ -1,3 +1,33 @@
 from django.shortcuts import render
-
+from rest_framework import generics
```

Comment:
Please implement one more custom endpoint to retrieve a list of departments with the average salary for these departments (response format should be json). This REST endpoint should be used to retrieve data for the Departments page.